shopBuddy/shopBuddy/Item+Persistent.h

```diff
@@ -0,0 +1,30 @@
+//
+//  Item+Persistent.h
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/21/22.
+//
+
+#import "Item.h"
+#import "Parse/Parse.h"
+#import "ShoppingList.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Item (Persistent)
+
+@property (nonatomic,copy) NSString *objectID;
+
++ (void) createObject:(PFObject *)object;
```

Comment:
You can delete this method since it does not involve any `Item` object in the implementation. You can call `[object saveInBackground]` directly in the callsites.